src/olympia/constants/webext_permissions.py

```diff
@@ -16,6 +16,7 @@
     u'http://*/*': ALL_URLS_PERMISSION,
     u'https://*/*': ALL_URLS_PERMISSION,
     u'*://*/*': ALL_URLS_PERMISSION,
+    # FIXME: how about https://*/path ?
```

Comment:
The entire file is going (no more constants, other than the all_urls special case) so...